mrmustard/lab_dev/wires.py

```diff
@@ -175,6 +175,10 @@ def __init__(
         # Adds elements to the cache when calling ``__getitem__``
         self._mode_cache = {}
 
+    def __len__(self) -> int:
+        r"The number of wires."
+        return sum(len(s) for s in self.args)
```

Comment:
wouldn't many wires be counted twice? not sure what our definition of a unique wire is

Reply:
why twice? When I define Wires with `Wires(a,b,c,d)` the total number of wires is `len(a) + len(b) + len(c) + len(d)`

Reply:
just my poor understanding of MrMustard :D I figured the modes_in and modes_out would be the same in some cases, hence the double counting, but it doesn't seem that's the case